.github/workflows/release.yml

```
@@ -0,0 +1,20 @@
+
+on:
+  #  push:
+  pull_request:
+    types:
+      - opened
+      - synchronize
+    branches:
+      - main
+
+jobs:
+  build:
```

actionlint

"steps" section is missing in job "build"

(syntax-check)